src/flagpole/flagpole_eval.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# flake8: noqa: S002
+
+import argparse
+import dataclasses
+import sys
+from pathlib import Path
+from typing import Any, Tuple, cast
+
+import yaml
+
+from flagpole import Feature
+from flagpole.conditions import Segment
+from flagpole.evaluation_context import EvaluationContext
+from sentry.utils import json
+
+
+def main() -> None:
+    args = get_arguments()
+
+    indent = "    "
+    print("Opening file:")
+    print(indent, args.get("flagpole_file"))
+    print("")
+    print("Evaluating flag:")
+    print(indent, args.get("flag_name"))
+    print("")
+    print("Evaluation Context:")
+
+    print(indent, args.get("context"))
+    print("")
+
+    try:
+        feature = read_feature(args.get("flag_name", ""), args.get("flagpole_file", ""))
+    except Exception as e:
```

Comment:
Is it worth catching by exception type for a better error message? Especially for yaml parsing or file opening issues

Example: if flagpole_file arg isn't a valid file, be more specific in error message indicating so